src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/Layers/ArcGISTiledLayer/metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+  "Name": "ArcGIS Tiled Layer",
```

Comment:
Oh yeah, I think we agreed to use lower cases in the names https://developers.arcgis.com/ios/beta/swift/sample-code/arcgis-tiled-layer-url-.htm